runtime/client_http_request.go

```diff
@@ -160,14 +160,22 @@ func (req *ClientHTTPRequest) WriteJSON(
 		httpReq.Header.Set(headerKey, headerValue)
 	}
 
+	acceptTypePresent := false
 	for k := range headers {
+		if k == "Accept" {
```

Comment:
Also, the header map stores these headers using textproto.MIMEHeader(val), you don't want to assume casing of the "Accept" constant instead compare:
```
const _acceptHeader = "accept"
...
if k == textproto.MIMEHeader(_acceptHeader)
```

Reply:
Great feedback, will get this in.